api/agent/call.go

```diff
@@ -83,6 +100,8 @@ func FromHTTPTriggerRequest(app *models.App, fn *models.Fn, trigger *models.Trig
 		// TODO this relies on ordering of opts, but tests make sure it works, probably re-plumb/destroy headers
 		// TODO async should probably supply an http.ResponseWriter that records the logs, to attach response headers to
 		if rw, ok := c.w.(http.ResponseWriter); ok {
+			// TODO deprecate after CLI is updated
+			rw.Header().Add("Fn-Call-ID", id)
```

Comment:
these end up being identical... they both produce `Fn-Call-Id`... do with that what you will